scripts/create-release-pr

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+CURRENT_BRANCH=`git branch --show-current`
+MAIN_BRANCH="master"
```

Comment:
made this variable to make it easier to switch our main branch on github from `master` to `main` when we do so